app/backend/bin/cli.py

```diff
@@ -0,0 +1,556 @@
+#!/usr/bin/env python
+
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', 'lib', 'boto'))
```

Comment:
Can you add:
sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', 'lib', 'celery'))

I am getting:
Traceback (most recent call last):
  File "./cli.py", line 26, in <module>
    import tasks
  File "./../tasks.py", line 14, in <module>
    from celery import Celery, group
ImportError: No module named celery